airflow/www_rbac/templates/appbuilder/baselayout.html

```diff
@@ -20,9 +20,9 @@
 
 {% block head_css %}
   {{ super() }}
-  <link href="{{ url_for('static', filename='main.css') }}" rel="stylesheet">
-  <link href="{{ url_for('static', filename='bootstrap-theme.css') }}" rel="stylesheet">
-  <link rel="icon" type="image/png" href="{{ url_for("static", filename="pin_30.png") }}">
+  <!--styleBundle.css file contains the styles from main.css and bootstrap-theme.css-->
```

Comment:
Again, HTML comment should be Jinja comment

Reply:
👍